common/src/Model/BrushBuilder.cpp

```diff
@@ -212,7 +212,7 @@ auto makeUnitCylinder(const size_t numSides, const RadiusMode radiusMode)
 }
 } // namespace
 
-Result<Brush> BrushBuilder::createCylinder(
+std::vector<Result<Brush>> BrushBuilder::createCylinder(
```

Comment:
Why does this return a vector now? If you want to use it for hollow cylinders, I suggest creating an overload instead. The signature is already too complicated.

Reply:
For reference CSG hollow brush right now sets brush thickness to whatever the grid is set to.